drm/radeon/: add Mullins UVD support.

Has same version of UVD as other CIK parts.

Signed-off-by: Samuel Li <samuel.li@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
diff -Naur a/drivers/gpu/drm/radeon/radeon_uvd.c b/drivers/gpu/drm/radeon/radeon_uvd.c
--- a/drivers/gpu/drm/radeon/radeon_uvd.c	2014-04-26 01:50:28.941068710 +0530
+++ b/drivers/gpu/drm/radeon/radeon_uvd.c	2014-04-26 02:15:33.177108251 +0530
@@ -98,6 +98,7 @@
 	case CHIP_BONAIRE:
 	case CHIP_KABINI:
 	case CHIP_KAVERI:
+	case CHIP_MULLINS:
 		fw_name = FIRMWARE_BONAIRE;
 		break;
 
